src/actions/index.js

```diff
@@ -1 +1,53 @@
-/*Put action creators here*/
\ No newline at end of file
+import {LOGIN, INVALID_CREDENTIALS, FETCHING, LOGOUT} from "./types";
+import axios from 'axios';
+
+const Login = (payload) => {
+    return {
+        type: LOGIN,
+        payload: payload
+    };
+};
+
+const InvalidCredentialsError = (payload) => {
+    return {
+        type: INVALID_CREDENTIALS,
+        payload: payload
+    }
+};
+
+const FetchAction = (payload) => {
+    return {
+        type: FETCHING,
+        payload: payload
+    }
+};
+
+const Logout = () => {
+    return {
+        type: LOGOUT
+    }
+};
+
+export const handleLoginResponse = (user_data) => async (dispatch) => {
+
+    // console.log("User data: ", user_data);
```

Comment:
Why do we have commented out code?

Reply:
Thanks for pointing that out, I've fixed it